larq_compute_engine/tflite/tests/bconv2d_test.cc

```diff
@@ -757,15 +1056,113 @@ TEST(BConv2DTests, BConvErrorTest) {
   EXPECT_DEATH(
       {
         Bitpacked_BConv2DOpModel m_lce(
-            compute_engine::tflite::Register_BCONV_2D64_OPT,
-            {TensorType_FLOAT32, {1, 16, 16, 64}},
-            {TensorType_FLOAT32, {128, 3, 3, 64}}, {TensorType_UINT8, {}}, 64,
+            compute_engine::tflite::Register_BCONV_2D64_OPT, input_tensor,
+            packed_filter_tensor, output_tensor, post_tensor, post_tensor, 64,
             1, 1, Padding_SAME, 0, ActivationFunctionType_NONE, 1, 1, false,
             true, 1);
       },
       "Writing bitpacked output is only supported with valid or one-padding.");
 }
 
+TEST(BConv2DTests, Int8ErrorTest) {
```

Comment:
we need to create a separate files for 8-bit bconv tests and a separate file for test helper functions and also a separate file for common things used for different tests (e.g, Bconv2DOpModel)